### ISSUE TITLE
Unable to Initialize plugin

### ISSUE THREAD

**Issue by fellowish** [TrevTV/mb_CopySpotify#4]
There is an issue with the plugin's installation and initialization. 

"Unable toinitialize plugin: mb_CopySpotify.dll
Unable to load one or more of the requested types."

And:

"Unable to save due to the following error:
Found invalid data while decoding."

**Comment by TrevTV** [comment]
Could you upload the error log from `MusicBee -> Help -> Support -> View Error Log`? Those messages don't have enough information.

**Comment by devmaxime** [comment]
Same behavior here, and cannot generate error log since no error was detected.

**Comment by TrevTV** [comment]
I just tested this plugin on a fresh MusicBee install and I got a `Unable to Initialize plugin` for a dependency library, but once I restarted MusicBee after, it worked perfectly fine.

**Comment by devmaxime** [comment]
I have a fresh MusicBee install on a win10 laptop that never had MusicBee. I tried to restart but sadly It didn't solve the issue.

Thank you for trying to help me using your plugin 👍. I'm going to try another way.

**Comment by vico93** [comment]
I have a similar issue with the plugin, here's what appears in ErrorLog.dat (take note that my MusicBee is in Portuguese):

```
11/08/2024 15:21:44 - 10.0.22631.0 - 3.5.8447.35843 Store - System.Reflection.ReflectionTypeLoadException: Não é possível carregar um ou mais dos tipos solicitados. Recupere a propriedade LoaderExceptions para obter mais informações.
   em System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   em System.Reflection.RuntimeModule.GetTypes()
   em System.Reflection.Assembly.GetTypes()
   em #=zqR2XciUj1qIekb1OHQ==..ctor(#=zjSxIWkhwlvsm$dC4uw== #=zQB6KZnw=)

11/08/2024 15:21:45 - 10.0.22631.0 - 3.5.8447.35843 Store - System.Reflection.ReflectionTypeLoadException: Não é possível carregar um ou mais dos tipos solicitados. Recupere a propriedade LoaderExceptions para obter mais informações.
   em System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   em System.Reflection.RuntimeModule.GetTypes()
   em System.Reflection.Assembly.GetTypes()
   em #=zqR2XciUj1qIekb1OHQ==..ctor(#=zjSxIWkhwlvsm$dC4uw== #=zQB6KZnw=)

11/08/2024 15:21:54 - 10.0.22631.0 - 3.5.8447.35843 Store - System.Reflection.ReflectionTypeLoadException: Não é possível carregar um ou mais dos tipos solicitados. Recupere a propriedade LoaderExceptions para obter mais informações.
   em System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   em System.Reflection.RuntimeModule.GetTypes()
   em System.Reflection.Assembly.GetTypes()
   em #=zqR2XciUj1qIekb1OHQ==..ctor(#=zjSxIWkhwlvsm$dC4uw== #=zQB6KZnw=)

11/08/2024 15:22:05 - 10.0.22631.0 - 3.5.8447.35843 Store - System.Reflection.ReflectionTypeLoadException: Não é possível carregar um ou mais dos tipos solicitados. Recupere a propriedade LoaderExceptions para obter mais informações.
   em System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   em System.Reflection.RuntimeModule.GetTypes()
   em System.Reflection.Assembly.GetTypes()
   em #=zqR2XciUj1qIekb1OHQ==..ctor(#=zjSxIWkhwlvsm$dC4uw== #=zQB6KZnw=)
```

**Comment by vico93** [comment]
I think i found the issue.

When installing the plugin via selecting the .zip file add plugin button on MusicBee seems like it didnt copy all the needed dlls to plugins folder, so the plugin cant be loaded. Unzipping manually to plugins folder makes it load correctly.

BUT, the fact that this plugin uses dlls like `Newtonsoft.Json.dll`, `EmbedIO.dll` and others and place them in the root `plugins` folder can clash with other plugins that uses different versions of those dlls like [Spotify Integration](https://www.getmusicbee.com/addons/plugins/185/spotify-integration-plugin/) so my suggestion is altering the code to load those dlls in a subfolder, like [DiscordBee](https://github.com/sll552/DiscordBee) does, avoiding such conflicts.

**Comment by TrevTV** [comment]
@vico93 Copied that approach and implemented it in [v1.2.0](https://github.com/TrevTV/mb_CopySpotify/releases/1.2.0).
